kafka/src/main/java/io/micronaut/configuration/kafka/annotation/ErrorStrategy.java

```diff
@@ -63,6 +68,16 @@
      */
     int retryCount() default DEFAULT_RETRY_COUNT;
 
+    /**
+     * Whether all exceptions should be handled or ignored when using RETRY_ON_ERROR and RETRY_EXPONENTIALLY_ON_ERROR
+     * {@link io.micronaut.configuration.kafka.annotation.ErrorStrategyValue}.
+     *
+     * By default, only the last failed attempt will be handed over to the exception handler.
+     *
+     * @return whether all exceptions should be handled or ignored
+     */
+    boolean handleAllExceptions() default DEFAULT_HANDLE_ALL_EXCEPTIONS;
```

Comment:
add `@since` to javadoc

Reply:
I just added `@since 5.0`. Please let me know if I should be targeting a different version.